tests/integration-tests/configs/develop.yaml

```diff
@@ -134,7 +135,7 @@ test-suites:
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
     test_cluster_custom_resource.py::test_cluster_delete_retain:
       dimensions:
-        - oss: ["alinux2"]
+        - oss: ["rocky8"]
```

Comment:
Not related to your changes but it would be good to modify all the existing custom_resource tests to run with different oses. Previously they ran just with alinux2, now you correctly added rocky8, in future we should enable other oses too.